source/index.html

```diff
@@ -1,12 +1,137 @@
 <!DOCTYPE html>
-<html lang="ru">
+<html class="page" lang="ru">
   <head>
     <meta charset="utf-8">
-    <title>HTML Academy: Седона</title>
+    <title>HTML Academy: Седона | Главная</title>
   </head>
-  <body>
+  <body class="page__body">
+    <header class="main-header">
+      <a class="main-header__logo" href="index.html">
+        <img src="img/city-of-sedona-logo.svg" width="185" height="96" alt="Логотип города Седона.">
+      </a>
 
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/adaptive">HTML и CSS. Адаптивная вёрстка и автоматизация</a>».</p>
+      <nav class="main-header__nav">
+        <button class="main-header__nav-toggle" type="button">
+          <span class="visually-hidden">Меню.</span>
+        </button>
+        <div class="nav-menu">
+          <ul class="nav-menu__list">
+            <li class="nav-menu__item">
+              <a class="nav-menu__link nav-menu__link--current" href="index.html">Главная</a>
+            </li>
+            <li class="nav-menu__item">
+              <a class="nav-menu__link" href="catalog.html">Фото и видео</a>
+            </li>
+            <li class="nav-menu__item">
+              <a class="nav-menu__link" href="form.html">Форма отзыва</a>
+            </li>
+            <li class="nav-menu__item">
+              <a class="nav-menu__link" href="https://htmlacademy.ru">HTML Academy</a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+    </header>
 
+    <main class="main-container">
+      <h1 class="visually-hidden">Седона.</h1>
+
+      <section class="background-image">
+        <h2 class="visually-hidden">Седона.</h2>
+        <img class="welcome-logo" src="img/welcome-logo.svg" width="282" height="214" alt="Добро пожаловать в прекрасный город Седона.">
+      </section>
+
+      <section class="description">
+        <h2 class="description__sedona">Седона — небольшой городок в Аризоне, заслуживающий большего!</h2>
+        <p class="description__reasons">Рассмотрим 5 причин, по которым Седона круче, чем Гранд-Каньон!</p>
+      </section>
+
+      <section class="advantages">
+        <h2 class="visually-hidden">Преимущества Седоны.</h2>
+
+        <ul class="advantages__list">
+          <li class="advantages__item advantages__item-with-photo">
+            <h3 class="advantages__title">Настоящий городок</h3>
+            <p class="advantages__number">&mdash; №1 &mdash;</p>
+            <p class="advantages__description">Седона — не аттракцион для туристов, там течёт своя жизнь</p>
+            <img class="sedona-city-photo" src="img/sedona-city-photo.jpg" alt="Седона.">
+            <ul class="advantages__item sublist">
```

Comment:
думаю неправильно задавать списку `advantages__item`, свойства отдельных преимуществ не подойдут списку
```suggestion
            <ul class="advantages__sublist sublist">
```